src/index.html

```diff
@@ -2,10 +2,76 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Wbpack Exercise</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="style.css" />
+    <!-- Bootstrap CSS -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+
+    <title>Leaderboard</title>
   </head>
   <body>
-    <h1>Helloo webpack!</h1>
+    <div class="container h-100">
+      <header class="header h1 mt-5">Leaderboard</header>
+      <div
+        class="main d-flex flex-row justify-content-between align-items-start"
+      >
+        <div class="scoreboard w-50">
+          <div class="recentScore d-flex justify-content-between">
+            <h2>Recent scores</h2>
+            <button class="btn btn-outline-dark">Refresh</button>
+          </div>
+          <table class="table border border-dark">
+            <thead class="d-none">
+              <tr>
+                <td>User</td>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>Name: 100</td>
+              </tr>
+              <tr>
+                <td>Name: 35</td>
+              </tr>
+              <tr>
+                <td>Name: 45</td>
+              </tr>
+              <tr>
+                <td>Name: 125</td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+        <div class="form w-50 h-100 d-inline-block">
+          <form class="w-75 m-auto">
+            <h2>Add your score</h2>
+
+            <div class="form-group">
+              <input
+                type="text"
+                class="form-control border border-dark"
+                id="name"
+                placeholder="Add your name"
+              />
+            </div>
+            <div class="form-group">
+              <input
+                type="text"
+                class="form-control border border-dark"
+                id="score"
+                placeholder="Add your score"
+              />
+            </div>
+
+            <button type="submit" class="btn btn-outline-dark">Submit</button>
+          </form>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] It looks like you create all the applications on `index.html`. On this script you must add something like `<div id="root"></div>`, then use javascript to take the created div and with js create all the application, in other words, is required don't use HTML code, in place, use JS code to make all the given template. 🐱 
- [ ] Don't forget to use ES6 best practices when you work on it. 🐶